src/test/java/net/snowflake/client/jdbc/HeartbeatIT.java

```diff
@@ -125,7 +118,7 @@ private void submitQuery(boolean useKeepAliveSession, int queryIdx)
   @Test
   @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
   public void testSuccess() throws Exception {
-    int concurrency = 5;
+    int concurrency = 10;
```

Comment:
why changed to 10?

Reply:
Looking through the history, I think it was originally supposed to be 10 and got changed to 5 at some point by accident. The comments that were there also said it is supposed to be 10 concurrent threads.